client/extensions/woocommerce/store-sidebar/store-ground-control.js

```diff
@@ -6,26 +6,34 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Gridicon from 'components/gridicon';
 import Site from 'blocks/site';
+import { canCurrentUserUseCustomerHome } from 'state/sites/selectors';
+import { getStatsDefaultSitePage } from 'lib/route/path';
 
-const StoreGroundControl = ( { site, translate } ) => {
+const StoreGroundControl = ( { canUserUseCustomerHome, site, translate } ) => {
 	const isPlaceholder = ! site;
-	const backLink = isPlaceholder ? '' : '/stats/day/' + site.slug;
+	const siteSlug = get( site, 'slug', '' );
+	const backDestination = canUserUseCustomerHome
```

Comment:
As there are a few places we're doing this, should we create a `getSiteAdminHomePageUrl` function?

Reply:
That's a great idea--will update.